pkg/kubectl/describe.go

```diff
@@ -2068,3 +2067,28 @@ func (fn typeFunc) Describe(exact interface{}, extra ...interface{}) (string, er
 	}
 	return s, err
 }
+
+// printLabelsMultiline prints multiple labels with a proper alignment.
+func printLabelsMultiline(out io.Writer, title string, labels map[string]string) {
```

Comment:
Are you talking about indentation between the begging of line and the first letter (`<1>`) or between title and label  (`<2>`)?

```
<1>Example:<2>foo=bar
<1>        <2>baz=quux
```